components/console/events.rst

```diff
@@ -178,16 +178,36 @@ Listeners receive a
         // gets the signal number
         $signal = $event->getHandlingSignal();
 
+        // sets the exit code
+        $event->setExitCode(0);
+
         if (\SIGINT === $signal) {
             echo "bye bye!";
         }
     });
 
+It is also possible to abort the exit if you want the command to continue its
+execution even after the event has been dispatched, thanks to the
+:method:`Symfony\\Component\\Console\\Event\\ConsoleSignalEvent::abortExit`
+method::
+
+    use Symfony\Component\Console\ConsoleEvents;
+    use Symfony\Component\Console\Event\ConsoleSignalEvent;
+
+    $dispatcher->addListener(ConsoleEvents::SIGNAL, function (ConsoleSignalEvent $event) {
+        $event->abortExit();
+    });
+
 .. tip::
 
     All the available signals (``SIGINT``, ``SIGQUIT``, etc.) are defined as
     `constants of the PCNTL PHP extension`_.
 
+.. versionadded:: 6.3
```

Comment:
lets move the version added above the tip please

Reply:
Done :slightly_smiling_face: